src/main/java/org/json/JSONObject.java

```diff
@@ -99,19 +101,28 @@ of this software and associated documentation files (the "Software"), to deal
      * whilst Java's null is equivalent to the value that JavaScript calls
      * undefined.
      */
-    private static final class Null {
+    private static final class Null{
 
         /**
          * There is only intended to be a single instance of the NULL object,
          * so the clone method returns itself.
          *
          * @return NULL.
          */
-        @Override
+    /*   @Override
```

Comment:
Why this code has been deactivated?

Reply:
Fixed in d3cd3f4c88c67b3621b6714e4af99b3048ad605d